File: python/func.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

if __name__ == "__main__":
	from func import fact
	import sys
	sys.stdout.write("Please enter an integer value: ")
	n = int(raw_input())
	sys.stdout.write("Fact(" + str(n) + ")= ")
	print(str(fact(n)))

def fact(i):
	if i <= 1:
		return 1
	return i * fact(i-1)

File: java/src/Func.java
import java.util.Scanner;
public class Func {
	public static void main(String[] args) {
		System.out.print("Please enter an integer value: ");
		Scanner cs = new Scanner(System.in);
		int n = cs.nextInt();
		System.out.print("Fact(" + n + ")=");
		System.out.println(fact(n));
	}

	public static int fact(int i){
		if (i <=1)
			return 1;
		return i * fact(i-1);
	}
}